.github/workflows/check.yml

```
@@ -0,0 +1,17 @@
+name: check
+on:
+  pull_request:
+  push:
+jobs:
+  pre-commit:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.5, 3.6, 3.7, 3.8]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v1
+      - run: pip install tox pre-commit
```

```suggestion
      - run: pip install tox
```
tox already installs pre-commit.